ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -95,17 +94,20 @@ const NetworksForm = ({
     label || (labelKey && t(getNetworkLabelKey(labelKey)));
   const [networkName, setNetworkName] = useState(selectedNetworkName || '');
   const [rpcUrl, setRpcUrl] = useState(selectedNetwork?.rpcUrl || '');
-  const [chainId, setChainId] = useState(selectedNetwork?.chainId || '');
+  const [chainId, setChainId] = useState(selectedNetwork?.caipChainId || ''); // ?
```

Comment:
this name is probably fine, right? since it is linked to the form field and not the network attribute